CHANGELOG.md

```diff
@@ -1,5 +1,56 @@
 # Change Log
 
+## Version 3.2.0 *(2020-04-02)*
+
+This is a new minor release with numerous internal changes.
+
+### Summary of changes
+
+* Chucker won't load the whole response into memory anymore, but will mutlicast it with the help of temporary files. It allows to avoid issues with OOM, like in this [issue](https://github.com/ChuckerTeam/chucker/issues/218).
+This change also allows to avoid problems with Chucker consuming responses, like reported in this [issue](https://github.com/ChuckerTeam/chucker/issues/242).
+* Added a red open padlock icon to clearly indicate HTTP requests in transactions list.
+* Added TLS info (version and cipher suite) into `Overview` tab.
+* Added ability to encode/decode URLs.
+* Added RTL support.
+* Switched from AsyncTasks to Kotlin coroutines.
+* Switched from Travis CI to Github Actions.
+* Switched to ViewBinding.
+* Bumped targetSDK to 29.
+* Greatly increased test coverage (we will add exact numbers and reports pretty soon).
+
+### Bugfixes
+
+* Fixed an [issue](https://github.com/ChuckerTeam/chucker/issues/218) with OOM exceptions on big responses.
+* Fixed an [issue](https://github.com/ChuckerTeam/chucker/issues/242) with Chucker throwing exception when used as `networkInterceptor()`.
```

Comment:
Maybe we should consolidate our link styles. Later in the CHANGELOG I was having links for PRs/Issues at the bottom of the doc so is easier to read when opening the RAW file:

```suggestion
* Fixed an issue [#218] with OOM exceptions on big responses.
* Fixed an issue [#242] with Chucker throwing exception when used as `networkInterceptor()`.
```

Here you're always highlighting the word `issue` so it makes hard to extract the link.

Reply:
I thought that just number won't get a link. Will fix it.